src/stream/mod.rs

```diff
@@ -25,7 +25,7 @@
 pub use empty::{empty, Empty};
 pub use once::{once, Once};
 pub use repeat::{repeat, Repeat};
-pub use stream::{Stream, Take};
+pub use stream::{Fuse, Stream, Take};
```

Comment:
Sticking with the `std` shouldn't this be `async_std::stream::FusedStream` as a counterpart to `std::iter::FusedIterator`? I know we haven't been really naming things that way, but figured it'd be worth bringing up.

cc/ @stjepang thoughts on naming?

Reply:
It should.

Reply:
FusedIterator is a trait,the struct is named Fuse: https://doc.rust-lang.org/std/iter/struct.Fuse.html.

Reply:
Ah my bad, thanks for catching that @matklad!